pom.xml

```diff
@@ -856,7 +856,7 @@
     <metrics-core.version>3.2.6</metrics-core.version>
     <!--
       Note that the version of jackson-[annotations,core,databind] must be kept in sync with the
-      version of jackson-jaxrs-json-provider shipped in hbase-thirdparty.
+      version of jackson-jax/rs-json-provider shipped in hbase-thirdparty.
```

Comment:
Why this change? I think 'jackson-jaxrs-json-provider' is the name of the library?

Reply:
Thanks.
That's just a typo.
Reverted.